packages/manager/modules/pci/src/projects/project/sidebar/project-list/project-list.controller.js

```diff
@@ -44,4 +63,21 @@ export default class ProjectListController {
         this.isLoading = false;
       });
   }
+
+  goToProjectDetails(project) {
+    return this.OvhApiCloudProjectServiceInfos.v6()
+      .get({ serviceName: project.project_id })
+      .$promise.then((serviceInfo) =>
+        project.status === CLOUD_PROJECT_STATE.suspended ||
+        serviceInfo.status === CLOUD_PROJECT_BILLING_STATE.pendingDebt
+          ? this.$state.go('pci.projects.project.inactive', {
```

Comment:
move $state to router

Reply:
project-list is just a re-usable component and it does not have it's own routes

Reply:
@ganeshkumar1989  maybe the link could be passed as a binding in that case, so the component can be reused in another context

Reply:
This is done. Thanks